### PR TITLE
Unneeded calls to get and get-in removed

### PR DESCRIPTION
**Pull request by jaor**

### CODE REVIEW

File: best-first/script.whizzml [hunk 1]
```diff
@@ -46,10 +45,10 @@
         make-req (lambda (mid) (assoc eval-req "model" mid))
         all-reqs (map make-req model-ids)
         evs (map fetch (create-and-wait* "evaluation" all-reqs))
-        vs (map (lambda (ev) (get-in ev ["result" "model" "average_phi"])) evs)
+        vs (map (lambda (ev) (ev ["result" "model" "average_phi"] 0)) evs)
         value-map (make-map potentials vs)
         max-val (get-max vs)
-        choose-best (lambda (id) (if (= max-val (get value-map id)) id false)))
+        choose-best (lambda (id) (and (= max-val (value-map id)) id)))
```
Comment (mmerce) [inline best-first/script.whizzml:51]: does (and true "something") return "something"?

Comment (whizzmler) [inline best-first/script.whizzml:51]: yes

File: gradient-boosting/script.whizzml [hunk 2]
```diff
@@ -42,21 +42,21 @@
 
 ;; Helper methods to get dataset attributes
 (define (get-fields dataset)
-  (get (fetch dataset {"limit" -1}) "fields"))
+  ((fetch dataset) "fields"))
```
Comment (mmerce) [inline gradient-boosting/script.whizzml:45]: Does (fetch dataset) retrieve all the fields structure? (no field number limits?). That's what limit -1 is here for, I guess...

File: gradient-boosting/script.whizzml [hunk 3]
```diff
@@ -76,7 +76,7 @@
 ;; the same fields to learn at each iteration.
 (define (get-inputs fields)
   (let (not-generated? (lambda (astr) (not (contains-string? boost-id astr)))
-        is-input? (lambda (fid) (not-generated? (get (get fields fid) "name"))))
+        is-input? (lambda (fid) (not-generated? ((fields fid) "name"))))
```
Comment (mmerce) [inline gradient-boosting/script.whizzml]: isn't this like?
(fields [fid "name])

**Comment by mmerce** [comment]
just a nit and some questions to ensure you're paying attention :-)

File: gradient-boosting/script.whizzml (hunk 2 above)
Comment (jaor) [inline gradient-boosting/script.whizzml:45]: yes

File: gradient-boosting/script.whizzml (hunk 3 above)
Comment (jaor) [inline gradient-boosting/script.whizzml]: indeed, fixed

**Pull request merged**